apps/hash-frontend/src/components/hooks/use-logout-flow.ts

```diff
@@ -35,7 +35,7 @@ export const useLogoutFlow = () => {
            * hook on the current page.
            */
           await router.push({
-            pathname: "/signin",
+            pathname: "/",
             query: { return_to: router.asPath },
           });
```

Comment:
Not caused by this change, but I noticed an edge case when logging out with the sidebar collapsed where the contents overflow the screen width for some reason. I assume some state is not being updated correctly when the sidebar is collapsed or something related. 

https://github.com/hashintel/hash/assets/42802102/e81ea887-1c53-4024-af4c-258b487a9287

Reply:
Thanks for flagging, have fixed in https://github.com/hashintel/hash/pull/4502/commits/b7beeba5d7b2485068787603479ecbefc433ee69